projects/jump-search/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Jump Search in Python
+layout: default
+date: 2020-10-02
+last-modified: 2020-10-02
+featured-image: jump-search-python.png
```

Comment:
I appreciate you including the image! Right now, all images should go into the `assets/sources/` folder. From there, our image-titler will actually edit the image for you. So, if you want to include an image, you can place a raw image in that folder.

Reply:
I don't have the raw image I only  have it in jpeg format

Reply:
Do you mean the image without text

Reply:
Yeah without text would be great.